tests/appsec/iast/sink/test_no_httponly_cookie.py

```diff
@@ -27,12 +27,12 @@ def setup_empty_cookie(self):
     def test_empty_cookie(self):
         self.assert_no_iast_event(self.request_empty_cookie)
 
-    @missing_feature(library="java", reason="Metrics implemented")
-    @missing_feature(library="python", reason="Metrics implemented")
-    @missing_feature(library="dotnet", reason="Metrics implemented")
+    @missing_feature(library="java", reason="Metrics not implemented")
```

Comment:
Maybe not for this PR, but in Java we should be covering metrics in all vulnerabilities related to response headers (they use the same code path)